### PR TITLE
bosun: print "UTC" next to the time when displaying graphs

### PR DESCRIPTION
**Pull request by nicollet**
partially solve issue 680
https://github.com/bosun-monitor/bosun/issues/680

The only relevant file changed is: cmd/bosun/web/static/js/directives.ts

### CODE REVIEW

File: cmd/bosun/main.go [hunk 1]
```diff
@@ -282,6 +282,10 @@ func main() {
 	}()
 
 	if *flagWatch {
+		// note for vim users, I've this in my .vimrc:
```
Comment (kylebrandt) [inline cmd/bosun/main.go]: Don't think this should be a comment in main.go. Maybe add a tips section to https://github.com/bosun-monitor/bosun/blob/master/CONTRIBUTING.md

File: cmd/bosun/web/static/js/bosun.js [hunk 2]
```diff
@@ -867,9 +867,9 @@ var UsernameInputController = (function () {
     function UsernameInputController(auth) {
         this.auth = auth;
     }
+    UsernameInputController.$inject = ['authService'];
```
Comment (kylebrandt) [inline cmd/bosun/web/static/js/bosun.js]: I'm not sure why this is moving. Which version of tsc are you using?

File: cmd/bosun/main.go (hunk 1 above)
Comment (nicollet) [inline cmd/bosun/main.go]: Sure, will update.

File: cmd/bosun/web/static/js/bosun.js (hunk 2 above)
Comment (nicollet) [inline cmd/bosun/web/static/js/bosun.js]: $ tsc --version
Version 2.4.2

Comment (kylebrandt) [inline cmd/bosun/web/static/js/bosun.js]: should be npm i -g typescript@2.3.1

Comment (kylebrandt) [inline cmd/bosun/web/static/js/bosun.js]: Then will need to run `go generate` again

**Comment by nicollet** [comment]
I think those changes should look good now. I don't know if I (or you) should rebase/push -f before applying. The last commit is not very worth it as it is just a go generate with the right version.

**Pull request merged**